app/editor/src/features/admin/reports/components/ReportSectionContent.tsx

```diff
@@ -156,6 +157,17 @@ export const ReportSectionContent = ({ index }: IReportSectionContentProps) => {
               }}
             />
           </Col>
+          <Checkbox
```

Comment:
We don't have a good toggle component for the Editor.  It would would take more time an effort to add one.  The checkbox does the trick.